lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -30,6 +30,20 @@ def self.checkout_and_pull(branch)
         return false
       end
 
+      # Create a new branch in preparation to do a hotfix.
+      #
+      # - Cuts the new branch from the tag `tag_version`
+      # - The name of the new branch will be `release/#{new_verison}`
+      #
+      # @param [String] tag_version The name of the tag to cut the hotfix from
+      # @param [String] new_version The name of the new version, e.g. "1.2.3"
+      #
+      def self.create_branch_for_hotfix(tag_version, new_version)
```

Comment:
Mentioned elsewhere already, but this seems to have overlap with `create_branch` – I wonder if there's some reuse possible here?

Reply:
Outdated diff, seems like you reviewed that code during a commit-by-commit review, but that is not supposed to be there anymore in the final PR diff 🤔